src/Octoshift/Commands/AbortMigration/AbortMigrationCommandArgs.cs

```diff
@@ -0,0 +1,26 @@
+﻿using OctoshiftCLI.Extensions;
+using OctoshiftCLI.Services;
+
+namespace OctoshiftCLI.Commands.AbortMigration;
+
+public class AbortMigrationCommandArgs : CommandArgs
+{
+    public string MigrationId { get; set; }
+    [Secret]
+    public string GithubPat { get; set; }
+
+    public const string REPO_MIGRATION_ID_PREFIX = "RM_";
+
+    public override void Validate(OctoLogger log)
+    {
+        if (MigrationId.IsNullOrWhiteSpace())
+        {
+            throw new OctoshiftCliException("MigrationId must be provided");
+        }
+
+        if (!MigrationId.StartsWith(REPO_MIGRATION_ID_PREFIX))
+        {
+            throw new OctoshiftCliException($"Invalid migration id: {MigrationId}");
```

Comment:
I would explain more why the migration id is invalid:
```suggestion
            throw new OctoshiftCliException($"Invalid migration id: {MigrationId}. Only repo migrations that start with \"RM_\" are currently supported.");
```

Reply:
Why are we assuming the error?